coordinator/store/wrapper/wrapper.go

```diff
@@ -22,6 +23,24 @@ import (
 	"github.com/edgelesssys/marblerun/coordinator/user"
 )
 
+// WrapTransaction initializes a transaction using the given handle,
+// and returns a wrapper for the transaction, as well as rollback and commit functions.
+func WrapTransaction(ctx context.Context, txHandle transactionHandle,
+) (wrapper Wrapper, rollback func(), commit func(context.Context) error, err error) {
+	tx, err := txHandle.BeginTransaction(ctx)
+	if err != nil {
+		return Wrapper{}, nil, nil, err
+	}
+	wrapper = New(tx)
+	rollback = func() {
+		tx.Rollback()
+	}
+	commit = func(ctx context.Context) error {
+		return tx.Commit(ctx)
+	}
+	return wrapper, rollback, commit, nil
```

Comment:
```suggestion
	return New(tx), tx.Rollback, tx.Commit, nil
```
AFAIK this is semantically equivalent